app/assets/javascripts/app/views/publisher_view.js

```diff
@@ -114,14 +114,9 @@ app.views.Publisher = Backbone.View.extend({
       form: form
     });
 
-    this.view_aspect_selector_blueprint = new app.views.PublisherAspectSelectorBlueprint({
-      el: this.$('.public_toggle > .dropdown'),
-      form: form
-    });
-
     this.view_getting_started = new app.views.PublisherGettingStarted({
       el_first_msg:  this.el_input,
-      el_visibility: this.$('.public_toggle > .dropdown'),
+      el_visibility: this.$('.public_toggle .aspect_dropdown > .dropdown-toggle'),
```

Comment:
Line is too long.<br>Identifier 'el_visibility' is not in camel case.<br>Mixed double and single quotes.<br>Too many errors. (22% scanned).